client/components/library.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import EmojiCard from './emojiCard';
+import LoadButton from '../components/load-button.jsx';
+
+export default function Library() {
+  const [emojis, setEmojis] = useState([]);
+  const [emojiIndex, setEmojiIndex] = useState(8);
+  const basePath = 'api/emojis';
+  useEffect(() => {
+    fetch(basePath, {
+      method: 'GET'
+    })
+      .then(res => res.json())
+      .then(data => {
+        setEmojis(data);
+      })
+      .catch(error => console.error(error));
+  }, []);
+
+  const library = emojis.map((emoji, index) => {
```

Comment:
If you wrote this as follows it would look better and might be small enough you could inline it below instead of using `{library}`:

```
emojis.map((emoji, index) => (index >= emojiIndex)
  ? <Fragment key={emoji.emojiId}>
  : <EmojiCard key={emoji.emojiId} name={emoji.name} url={emoji.url} />);